src/frontend/components/QuillEditor/ToolbarSetup.js

```diff
@@ -0,0 +1,49 @@
+// Modules object for setting up the Quill editor
+const modules = {
+  toolbar: [
+    [{ font: [] }],
+    [{ size: ["small", false, "large", "huge"] }], // custom dropdown
+    ["bold", "italic", "underline", "strike"], // toggled buttons
+    [{ header: [1, 2, 3, 4, 5, 6, false] }],
+    [{ list: "ordered" }, { list: "bullet" }],
+    [{ indent: "-1" }, { indent: "+1" }], // outdent/indent
+    [{ script: "sub" }, { script: "super" }], // superscript/subscript
+    // [({ header: 1 }, { header: 2 })], // custom button values
+    ["blockquote", "code-block"],
+    [{ direction: "rtl" }], // text direction
+    [{ align: [] }],
+    [{ color: [] }, { background: [] }], // dropdown with defaults from theme
+    ["link", "image", "video"],
+    ["formula"],
+    ["clean"],
```

Comment:
I like your commented line of code 
this is helping to understand the in easy way